website/src/pages/use-cases/code-reuse.tsx

```diff
@@ -0,0 +1,298 @@
+                            <li>
+                                Without easy access to existing patterns, developers default to implementing from
+                                scratch, unintentionally increasing complexity and confusion in the codebase.
+                            </li>
+                            <li>
+                                Engineering leaders lack the visibility needed to ensure teams adopt recommended
+                                packages and retire deprecated ones.
+                            </li>
+                        </ul>
+                    </div>
+                    <div className="col-lg-5 mt-8 text-center">
+                        <section className="case-studies__quote--in-content--section font-weight-normal">
+                            <h4 className="mb-3">FactSet ensures consistency across its entire codebase</h4>
+                            <p>
+                                “If I’m developing code for a library that might draw charts, for example, we don’t want
+                                30 different ways to draw a chart at FactSet. With Sourcegraph, I can search the code to
+                                find other chart examples, and simply copy the code. This saves us time and ensures
+                                consistency.”
+                            </p>
+                            <figcaption className="pt-2 text-muted text-center">
+                                &mdash; Joseph Majesky, Software Engineer at FactSet
+                            </figcaption>
+                        </section>
+                        <img className="my-4" src="/external-logos/factset-logo.svg" width="150" alt="Factset logo" />
+                        <Link className="d-block font-weight-bold" to="/blog/log4j-log4shell-0-day">
+                            Read the case study
+                            <ArrowRightIcon className="ml-2" />
+                        </Link>
+                    </div>
+                </div>
+            </ContentSection>
+        </div>
+
+        <ContentSection>
+            <div className="row mt-lg-0 mt-5 justify-content-center">
+                <div className="d-flex flex-column mt-lg-6 mt-4 w-100 mx-3">
+                    <h1 className="font-weight-bold text-lg-center text-left mb-lg-6 mb-md-6">How Sourcegraph helps</h1>
+                </div>
+                <div className="pb-lg-5 pb-md-6 pb-5">
+                    <CustomCarousel items={items} autoAdvance={true} smallPanel={true} />
+                </div>
+            </div>
+        </ContentSection>
+
+        <div className="bg-light-gray-3">
+            <ContentSection>
+                <div className="row d-flex flex-column mx-4 mx-lg-0 py-7 align-items-lg-center align-items-left">
+                    <div className="mb-5 d-flex flex-column text-center max-w-600">
```

Comment:
Since you have a max width on this container, and it has the potential of being smaller than its parent container, we want to center this as the child but keep the text aligned left. So in this case you'll need to add `mx-auto` to the div.

Reply:
I don't understand the ask. This is centered on larger screens & left aligned, centered container on mobile right now? The width isn't 600 on mobile, I tested on the smallest screen to check. Would you like me to add a breakpoint for the max-width for an added assurance? Adding `mx-auto` doesn't give the desired spacing in my troubleshooting. Why abandon this method if it's working?

Reply:
It's a subtle discrepancy but only noticeable between `md` and `lg`.  [I recorded a loom](https://www.loom.com/share/1d307b61ec264be6ad990021474f12dc) so hopefully that helps.

Reply:
🤯  so subtle! Thanks for the video! Should be good now 👍